### DISCUSSION TITLE
Nuxt 3: Serverless Function has timed out

### DISCUSSION THREAD

**Discussion by treighmawaka** [vercel/community#4122] [Help]
### Summary

I have been trying to deploy an update to our `Nuxt 3` app on Vercel on the staging repo (next) for the past few hours, but all attempts are failing. I'm getting the following `504` error in the frontend `This Serverless Function has timed out.` The app runs well locally.

I’ve tried deleting unused api routes, redeploying, updating dependencies, etc. But, nothing gives.

## My environment:

nuxt: 3.7.3
deployment branch: next
deployment domain: next.kinspots.co
plan: PRO

### Example

kinspots-git-next-kinspots.vercel.app

### Steps to Reproduce

Run the temporary deployment at kinspots-git-next-kinspots.vercel.app to see the error

**Comment by SteveChurch** [comment]
Did you ever resolve this? I have been looking for a solution, I've seen plenty of docs around using the default Next router but nothing more focused at using Nuxt. Defining the path to the server enpoints doesn't seem to work either.
